src/service/company.js

```diff
@@ -70,10 +70,15 @@ const updateCompany = async (company) => {
 
 const getShowcase = async () => await service.get('/showcase');
 
+const getDiscountRequest = async (id) => {
+  return service.get(`/companies/${id}/discount_requests`);
```

Comment:
I think you could change from `id` to `companyId` here so that it will be explicit that we need the company's ID:
```suggestion
const getDiscountRequest = async (companyId) => {
  return service.get(`/companies/${companyId}/discount_requests`);
```